app/models/manageiq/providers/openshift/container_manager_mixin.rb

```diff
@@ -84,6 +84,17 @@ def connect_client(kind, api_version, method_name)
     end
   end
 
+  OPENSHIFT_ROUTES = {
+    "hawkular"          => %w[hawkular-metrics openshift-infra],
+    "prometheus"        => %w[prometheus openshift-metrics],
+    "prometheus_alerts" => %w[alerts openshift-metrics]
+  }.freeze
```

Comment:
Note: these projects and route names aren't the same on openshift v4, still working on that part but at least this won't throw an exception in the UI